storybook/story-components/doc-code.js

```diff
@@ -1,5 +1,21 @@
-import { createElement } from 'react-native';
+import React from 'react';
+import { Text, StyleSheet } from 'react-native';
+import { size, fontFamily } from './platform-styles';
 
-const DocCode = () => createElement('code');
+const removeStartAndEndNewLines = str => str.replace(/^\s+|\s+$/g, '');
```

Comment:
Is this to remove whitespace? I see it's not using the multiline flag so wouldn't `str.trim()` achieve the same thing?

Reply:
👍  Doh! Good catch. At first I was just trimming the new line and not the white space. `str.trim()` is much cleaner!

Reply:
Easy to get caught up 👍  Thought I'd check in case I missed something 😄